### PR TITLE
[9989] Add scyllaDB mention to documentation

### PR DESCRIPTION
**Pull request by andrii-kysylevskyi**
Updated documentation according to the original issue available [here](https://github.com/celery/celery/issues/9989).

I have also considered adding testing infrastructure for both Cassandra and Scylla, but decided to keep the changes to a minimum. Would be happy to change that, though and to add tests as well, if maintainers consider this valuable.

### CODE REVIEW

File: celery/app/backends.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     'database': 'celery.backends.database:DatabaseBackend',
     'elasticsearch': 'celery.backends.elasticsearch:ElasticsearchBackend',
     'cassandra': 'celery.backends.cassandra:CassandraBackend',
+    'scylladb': 'celery.backends.cassandra:CassandraBackend',
```
Comment (auvipy) [inline celery/app/backends.py:26]: this is not a documentation change anymore with this change....
